src/main/kotlin/calculator/Operators.kt

```diff
@@ -0,0 +1,16 @@
+package calculator
+
+object Operators {
+    private val operators: MutableMap<String, Operator> = HashMap()
+
+    init {
+        operators["+"] = Operator.PLUS
+        operators["-"] = Operator.MINUS
+        operators["*"] = Operator.MULTI
+        operators["/"] = Operator.DIVIDE
+    }
+
+    fun getOperator(stringOperate: String): Operator {
+        return operators[stringOperate] ?: throw IllegalArgumentException()
```

Comment:
```suggestion
        return operators[stringOperate] ?: throw IllegalArgumentException("+,-,/,* 연산만 가능합니다.")
```

operator에 포함되지 않은 경우의 메시지 처리가 필요해 보입니다 😁